recipes/ptvsd/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "ptvsd" %}
+{% set version = "4.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: cfcde6a3de3cfa720e4f637af13deeae744f6dc6665b9bda92380885caf16ae6
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - pytest-runner
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - ptvsd
+
+about:
+  home: https://github.com/Microsoft/ptvsd/
+  license: MIT
```

Comment:
This is wrong. pydevd is licensed under EPL which makes the combined work EPL in my understanding

Reply:
I just got that from the [`LICENSE`](https://github.com/Microsoft/ptvsd/blob/master/LICENSE) file. I'm happy to defer to whatever is the correct thing to do here though...

Reply:
The EPL license is [included in the source](https://github.com/Microsoft/ptvsd/blob/master/src/ptvsd/_vendored/pydevd/LICENSE) - just reading it now...

Reply:
Take a look a this:

https://github.com/Microsoft/ptvsd/blob/master/src/ptvsd/ThirdPartyNotices.txt

Reply:
[Seems relevant](https://github.com/Microsoft/ptvsd/blob/master/src/ptvsd/_vendored/pydevd/LICENSE#L89-L94):
> When the Program is made available in source code form:
> 
>   a) it must be made available under this Agreement; and
>   b) a copy of this Agreement must be included with each copy of the Program.
>      Contributors may not remove or alter any copyright notices contained within the Program.